src/index.html

```diff
@@ -1,17 +1,192 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
+    <script type="text/javascript" src="./main.js"></script>
   </head>
+
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form onsubmit="onSubmit()">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <div class="form_field">
+          <label for="Surname">
+            Surname:
+            <input
+            type="text"
+            name="Surname"
+            id="Surname"
+            autocomplete="off">
```

Comment:
```suggestion
            <input
              type="text"
              name="Surname"
              id="Surname"
              autocomplete="off"
            >
```
fix attribute indents in all cases